paddle/fluid/operators/sequence_ops/sequence_concat_op.h

```diff
@@ -73,16 +73,23 @@ class SeqConcatKernel : public framework::OpKernel<T> {
     for (auto &x : xs) {
       if (lod_size == 0) {
         PADDLE_ENFORCE_EQ(x.get().lod().empty(), false,
-                          "Input(X) Tensor of SequenceConcatOp does not "
-                          "contain LoD information.");
+                          platform::errors::InvalidArgument(
+                              "Input(X) Tensor of SequenceConcatOp does not "
+                              "contain LoD information."));
         lod_size = x.get().lod()[0].size();
       } else {
-        PADDLE_ENFORCE_EQ(
-            lod_size, x.get().lod()[0].size(),
-            "The number of sequence must be same between each input");
+        PADDLE_ENFORCE_EQ(lod_size, x.get().lod()[0].size(),
+                          platform::errors::InvalidArgument(
+                              "The number of sequence must be same between "
+                              "each input. But received (%d) != (%d)",
+                              lod_size, x.get().lod()[0].size()));
       }
     }
-    PADDLE_ENFORCE_NE(lod_size, 0, "Each input must have sequence information");
+    PADDLE_ENFORCE_NE(lod_size, 0,
+                      platform::errors::InvalidArgument(
+                          "Each input must have sequence information. But "
+                          "received input lod size is(%d)",
```

Comment:
user may be confused about the relation between `sequence info` and `lod size`, we can just tell users no sequence information here